Dockerfile.tor-onionbalance-manager

```diff
@@ -7,16 +7,22 @@ ARG TARGETOS TARGETARCH
 RUN --mount=target=. \
     --mount=type=cache,target=/root/.cache/go-build \
     --mount=type=cache,target=/go/pkg \
-    CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -a -o /out/onionbalance-local-manager ./agents/onionbalance/main.go
+    CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -a -ldflags="-s -w" -o /out/onionbalance-local-manager ./agents/onionbalance/main.go
 
-FROM alpine:3.16.2
+FROM docker.io/library/alpine:3.17.3
```

Comment:
Hi @lexfrei, please check in alpine:3.17.3 we no longer face the issue described at https://github.com/bugfest/tor-controller/issues/30

Reply:
Deployed with [this manifest](https://github.com/lexfrei/k8s/blob/master/manifests/me-site/me-site.yaml)
Handled by [this chart](https://github.com/lexfrei/k8s/blob/master/helmfile.d/network.yaml#L83-L84) (only armv8 image is available)